File: go/1035.go
// Funcional no beecrowd!
package main

import (
	"fmt"
)

func main() {
	var A int
	var B int
	var C int
	var D int

	fmt.Scan(&A)
	fmt.Scan(&B)
	fmt.Scan(&C)
	fmt.Scan(&D)

	AB := A + B
	CD := C + D

	if B > C && D > A && AB < CD && C > 0 && D > 0 && A%2 == 0 {
		fmt.Println("Valores aceitos")
	} else {
		fmt.Println("Valores nao aceitos")
	}

}

File: Lua/Iniciante/1035.lua
-- Funcional no beecrowd
local A = io.read("n")
local B = io.read("n")
local C = io.read("n")
local D = io.read("n")

local AB = A + B
local CD = C + D

if B > C and D > A and AB < CD and C > 0 and D > 0 and A %2 == 0 then
    print("Valores aceitos")
else
    print("Valores nao aceitos")
end